Coverage howto docs (#137)
Clinical-Genomics/chanjo2: docs/usage/sample-coverage.md

## Coverage queries on d4 files or data contained in the database

> Chanjo2 can be used to quickly access average coverage depth statistics for an interval of a list of genomic intervals on a custom d4 coverage file.

Reviewer: ```suggestion
Chanjo2 can be used to quickly access average coverage depth statistics for an interval from a list of genomic intervals on a custom d4 coverage file.
```

> Otherwise Samples stored in the database contain the location or d4 files on disk or on a remote server. For this reason it is possible to retrieve coverage statistics relative to one or more samples by specifying their name (or case name).

Reviewer: ```suggestion
Additionally, samples stored in the database contain the location of d4 files on disk or on a remote server. For this reason, it is possible to retrieve coverage statistics relative to one or more samples by specifying their name (or the case name).
```

### Direct coverage query over a single genomic interval

Coverage over a genomic interval can be computed by sending a `GET` request to the `/coverage/d4/interval/` endpoint.

The parameters accepted by this query are:

- <strong>coverage_file_path</strong> (required). Path to a d4 file that is stored on the local drive or on a remote server (slower computation)
- <strong>chromosome</strong> (required). Genomic interval chromosome
- <strong>start</strong>. Genomic interval start position 
- <strong>stop</strong> Genomic interval end position 

Note that start and stop coordinates are not required and whenever they are omitted coverage statistics will be computer over the entire chromosome.

Query example:

``` shell
curl -X 'GET' \
  'http://localhost:8000/coverage/d4/interval/?coverage_file_path=https%3A%2F%2Fd4-format-testing.s3.us-west-1.amazonaws.com%2Fhg002.d4&chromosome=7&start=124822386&end=124929983' \
  -H 'accept: application/json'
```

> This query wll return a response like this, where the mean coverage over the single interval is present in the first element of the mean_coverage value:

Reviewer: ```suggestion
This query wll return a response, where the mean coverage over the single interval is present in the first element of the mean_coverage value:
```
